pyQuARC/schemas/checks.json

```
@@ -238,5 +243,15 @@
         "data_type": "custom",
         "check_function": "get_data_url_check",
         "available": true
+    },
+    "granule_project_short_name_check": {
+        "data_type": "string",
+        "check_function": "granule_project_short_name_check",
+        "available": true
+    },
+    "granule_sensor_short_name_check": {
+        "data_type": "string",
+        "check_function": "granule_sensor_short_name_check",
+        "available": true
     }
-}
+}
```

```suggestion
}

```
Recommend adding a new line at the end of a file